src/components/Invoice-list.tsx

```diff
@@ -0,0 +1,64 @@
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import EditIcon from '@mui/icons-material/Edit';
+import DeleteIcon from '@mui/icons-material/Delete';
+import IconButton from '@mui/material/IconButton';
+import { Grid } from '@mui/material';
+
+function createData(invoiceNumber: string, created: string, validUntil: string, amount: number, actions: null) {
+  return { invoiceNumber, created, validUntil, amount, actions };
+}
+
+const rows = [
+  createData('21/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('22/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('23/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('24/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('25/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+];
+
+export default function InvoiceList() {
+  return (
+    // Skąd lepiej importować? Bardziej czy mniej szczegółowo?
+    <Grid container spacing={2} p={4} direction={'column'} mb={0}>
+      <TableContainer component={Paper}>
+        <Table sx={{ minWidth: 650 }} aria-label="simple table">
```

Comment:
I think the table should have a valid label :D

Reply:
What do you mean by valid label?

Reply:
`Simple table` doesn't seem to be descriptive, should be probably more like a `Invoices table` or something like that.